python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -297,3 +305,38 @@ def sort_values(
         df4.divisions = tuple(methods.tolist(divisions))
 
     return df4
+
+
+# Define _get_shuffle_type utility to set the shuffle-kwarg
+# default and to validate user-specified options
+#
+# Supported Options:
+#  - "tasks"
+#  - "explicit-comms"  (requires dask_cuda)
+try:
+    import dask_cuda  # noqa: F401
+
+    def _get_shuffle_type(shuffle):
+        shuffle = shuffle or dask.config.get("shuffle", "tasks")
+        if shuffle not in {"tasks", "explicit-comms"}:
+            raise ValueError(
+                f"Dask-cudf only supports in-memory shuffling with "
+                f"'tasks' or 'explicit-comms'. Got shuffle={shuffle}"
+            )
+        return shuffle
```

Comment:
@wence- this change seems fine to me, but it does mean we will now **always** try to import dask-cuda when we import `dask_cudf.sorting`.  This introduces a bit more overhead to the the case that explicit-comms is **not** being used than trying to import dask-cuda multiple times introduces to the case that explicit-comms **is** being used (since the imports are cached).

Reply:
Since this is top-level in the file, we only pay the cost once (at first import of `dask_cudf`), so I don't think this is really bad.

Reply:
>I don't think this is really bad

Agree that it is not very significant. I just wanted to clarify that we are *slightly* improving the `shuffle="explicit-comms"` scenario at the expense of a slight regression in the more-common `shuffle="tasks"` scenario (at least that is how I understand it).

Reply:
Ah, I see what you mean, because now (when `import dask_cuda` succeeds) we have a few more conditions in the call to `rearrange_by_column`. If that ever becomes a performance problem, I will be delighted :).

Reply:
Note that I rolled back this change (while fixing the test) because my gut is telling me to avoid a top-level `dask_cuda` import.  However, I will be happy to revise if you have a strong inclination that it is the better way to go.

Reply:
>Ah, I see what you mean, because now (when import dask_cuda succeeds) we have a few more conditions in the call to rearrange_by_column.

I am actually referring to the fact that we now need to wait for `import dask_cuda` (which can take multiple seconds), even if we are not using "explicit-comms", and do not need to import `dask_cuda`.